airflow/www/static/js/trigger.js

```diff
@@ -20,13 +20,32 @@
 /* global document, CodeMirror, window */
 
 const textArea = document.getElementById('json');
+const recentConfigList = document.getElementById('recent_configs');
 const minHeight = 300;
 const maxHeight = window.innerHeight - 450;
 const height = maxHeight > minHeight ? maxHeight : minHeight;
 
 CodeMirror.fromTextArea(textArea, {
   lineNumbers: true,
-  mode: { name: 'javascript', json: true },
+  mode: {
+    name: 'javascript',
+    json: true,
+  },
   gutters: ['CodeMirror-lint-markers'],
   lint: true,
-}).setSize(null, height);
+})
+  .setSize(null, height);
+
+function setRecentConfig(e) {
+  let { value } = e.target;
+  try {
+    const json = JSON.parse(value);
+    value = JSON.stringify(json, null, 2);
+  } catch (err) {
+    // any error is non-json parseable
+    value = 'Error: config could not be parsed';
```

Comment:
We can throw a `console.error()` here, but lets leave the value as a single string if we can't parse it.

Reply:
done